integration-tests/emulated/src/tests/spiritnet/teleportation.rs

```diff
@@ -0,0 +1,56 @@
+use crate::{
+	mock::{
+		network::MockNetworkPolkadot,
+		para_chains::{AssetHubPolkadot, Spiritnet},
+		relay_chains::Polkadot,
+	},
+	utils::get_account_id_from_seed,
+};
+use frame_support::assert_err;
+use frame_support::dispatch::RawOrigin;
+use integration_tests_common::{asset_hub_polkadot, polkadot::ED, ALICE, BOB};
+use sp_core::sr25519;
+use sp_runtime::{DispatchError, ModuleError};
+use spiritnet_runtime::PolkadotXcm as SpiritnetXcm;
+use xcm::v3::WeightLimit;
+use xcm_emulator::{Here, Junction, Junctions, ParentThen, TestExt, X1};
+
+#[test]
+fn test_teleport_asset_from_regular_spiritnet_account_to_asset_hub() {
+	MockNetworkPolkadot::reset();
+
+	let alice_account_id = get_account_id_from_seed::<sr25519::Public>(ALICE);
+	let bob_account_id = get_account_id_from_seed::<sr25519::Public>(BOB);
+
+	Spiritnet::execute_with(|| {
+		assert_err!(
```

Comment:
This should also be `assert_noop` probably.

Reply:
yeah would be better: [4d6b9af](https://github.com/KILTprotocol/kilt-node/pull/603/commits/4d6b9af30851c78404e5d711ffaae561188fef33)